pyproject.toml

```diff
@@ -72,3 +72,7 @@ build_dir = "jupyterlab_mystjs/labextension"
 
 [tool.jupyter-releaser.options]
 version_cmd = "hatch version"
+
+[tool.jupyter-releaser.hooks]
+before-build-npm = ["python -m pip install jupyterlab~=3.1", "jlpm", "jlpm build:prod"]
```

Comment:
should we also be cleaning before this step to avoid the dupe files?

Reply:
`build:prod` should clean anything that is already there, so it should be fine? At this stage, I'm mainly hacking in the background of another task :)